share/static/html/page.html

```diff
@@ -107,7 +107,7 @@ <h1 class="mdl-typography--display-2">
   {% block footer %}
   <footer class="mdl-mini-footer everware-footer">
       <div class="mdl-mini-footer__left-section">
-        Powered by <a href="https://jupyter.org/">Jupyter</a> and <a href="https://www.docker.com/">Docker</a>
+        v0.5.0 Powered by <a href="https://jupyter.org/">Jupyter</a> and <a href="https://www.docker.com/">Docker</a>
```

Comment:
I agree.  The main problem was about render() of spawn.html. It isn't rendered in everware's code. Anyway I ll look again, what I can do.